man: add more info link and --locale example (#5504)
tldr-pages/tldr: pages/common/man.md

# man

> Format and display manual pages.

> > More information: <https://www.man7.org/linux/man-pages/man1/man.1.html>

Reviewer: ```suggestion
> More information: <https://www.man7.org/linux/man-pages/man1/man.1.html>.
```

`man -w {{command}}`

> - Display the man page for a custom overrided locale (L):

Reviewer: ```suggestion
- Display the man page using a specific locale:
```

> `man {{command}} -L {{locale}}`

Reviewer: We prefer long argument format when is available:
```suggestion
`man {{command}} --locale={{locale}}`
```
(you can edit the other examples BTW)

Reviewer 2: Should we have users perform this sort of update when they're just translating? @sbrl

Reviewer: :eyes: this isn't a translation PR. I didn't get it.

Reviewer 2: Nevermind, I didn't even read this PR's title, I just assumed it was a translation 😂

Author: @bl-ue lol, I was just thinking may be I could also give other contribution than translation ^^
@navarroaxel : Ok for the long version of the options, I was not aware of the convention, my mistake

Reviewer: np @patricedenis, it's not explicit in the contribution guidelines, I think.
